### PR TITLE
Added Dialtone 5 cross-link, top-banner

### PR DESCRIPTION
**Pull request by juliodialpad**
## Description
Version 6 Documentation needs a cross-link allowing users to get to version 5 documentation.

https://switchcomm.atlassian.net/browse/DT-101?atlOrigin=eyJpIjoiNGIyOGE5NGYzOTY2NDRhYjg4OTBlMDNiYjhiNjZjMWYiLCJwIjoiaiJ9
 
## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [x] Update, remove, or extend all affected documentation.
 - [x] Provide a description what is being changed, extended, or removed.
 - [x] Link to any issue(s) this request is resolving.
 - [x] Request a review from Drew Chandler, Joshua Hynes, or Ted Goas.

### CODE REVIEW

File: docs/_includes/banner.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<aside class="d-banner d-banner--info d-ps-relative js-dialtone5-banner" role="alert" aria-hidden="false">
+  <div class="d-banner__dialog" role="alertdialog" aria-labelledy="info-alert-title" aria-describedby="info-alert-desc">
```
Comment (dialpad-drew) [inline docs/_includes/banner.html]: Can we add `.d-w-unset` to this div so the width collapses to the content?

**Review by dialpad-drew — APPROVED** [review]
Approved with one piece of feedback. @ryandialpad is probably better at providing feedback on the scripting.

File: docs/assets/js/banner.js [hunk 2]
```diff
@@ -55,6 +56,9 @@ $(document).ready(function() {
             banner.addClass('d-banner--pinned');
             actions.addClass('d-d-none');
             topNav.attr('style','height: 11.2rem !important; padding-top: 4.8rem; max-height: 11.2rem !important;');
+            sideNav.attr('style','top: 6.4rem!important');
+            dialtone5Banner.addClass('d-d-none');
+            console.log('here');
```
Comment (dbecherdialpad) [inline docs/assets/js/banner.js]: you can remove this

Comment (juliodialpad) [reply docs/assets/js/banner.js]: sorry I missed that one

**Pull request merged**